homeassistant/components/samsungtv/device_trigger.py

```diff
@@ -65,7 +72,8 @@ async def async_attach_trigger(
     trigger_info: TriggerInfo,
 ) -> CALLBACK_TYPE:
     """Attach a trigger."""
-    if (trigger_type := config[CONF_TYPE]) == TURN_ON_PLATFORM_TYPE:
+    trigger_type = config[CONF_TYPE]
+    if [TURN_ON_PLATFORM_TYPE, TURN_OFF_PLATFORM_TYPE].count(trigger_type) == 1:
```

Comment:
Again should be `config[CONF_TYPE] in [TURN_ON_PLATFORM_TYPE, TURN_OFF_PLATFORM_TYPE]`.